src/test/configs/example/.devcontainer.json

```diff
@@ -3,8 +3,16 @@
 {
 	"image": "mcr.microsoft.com/devcontainers/base:latest",
 	"features": {
-		"ghcr.io/devcontainers/features/go:1": {
+		"ghcr.io/devcontainers/features/python:1": {
 			"version": "latest"
 		}
+	},
+	"postStartCommand": {
+		"poetry setup": [
```

Comment:
Why did we update this? The tests using this config would fail as python Feature doesn't install poetry.

Reply:
reverted